Playbooks/AS-PagerDuty-Integration/README.md

```diff
@@ -0,0 +1,136 @@
+<img src="https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/1-CONTRIBUTION-GUIDE/images/deploytoazuregov.png"/>
+</a>
+
+
+This playbook is intended to be run from a Microsoft Sentinel incident. It will create a PagerDuty event with the related Microsoft Sentinel incident and entity information.
+                                                                                                                                     
+![PagerDuty_Demo](Images/PagerDuty_Demo.png)
+ 
+                                                                                                                                     
+#
+### Requirements
+                                                                                                                                     
+The following items are required under the template settings during deployment: 
+
+* A PagerDuty integration key
+                                                                                                                                    
+# 
+### Setup
+                                                                                                                                     
+To Create a PagerDuty Integration Key:
+ 
+Log into your PagerDuty account and under the "**Services**" menu option, select "**Service Directory**". From the page, click the "**New Service**" button.
+ 
+![PagerDuty_Create_Integration_Key_1](Images/PagerDuty_Create_Integration_Key_1.png)
+
+                                                                                                       
+Add a Name and optional Description like the following:
+
+![PagerDuty_Create_Integration_Key_2](Images/PagerDuty_Create_Integration_Key_2.png)              
+                                                                                       
+                                                                                         
+Select an escalation policy for your service.
+
+![PagerDuty_Create_Integration_Key_3](Images/PagerDuty_Create_Integration_Key_3.png)
+
+                                                                                           
+Finally, select the Microsoft Azure option under the Integrations section, then click "**Create Service**".
+
+![PagerDuty_Create_Integration_Key_4](Images/PagerDuty_Create_Integration_Key_4.png)
+
+                                                                                                  
+You will be redirected to the service you have created. Under the Integrations tab view, you will find your integration key. This will be needed for the deployment of this playbook.
+                                                                                                  
+![PagerDuty_Create_Integration_Key_5](Images/PagerDuty_Create_Integration_Key_5.png)
```

Comment:
Can we highlight from where we have to fetch the integration key. Maybe encircle the block with a red box.

Reply:
Thank you for catching this. This is resolved.